src/components/Scanner.vue

```diff
@@ -6,41 +6,86 @@
       </ion-button>
     </ion-buttons>  
   </ion-toolbar>   
-  <div class="scanner">
-    <StreamBarcodeReader
+  <ion-content class="content">
+    <div class="scanner">
+      <StreamBarcodeReader
       @decode="onDecode"
-    />
-  </div> 
+      />
+    </div> 
+    <ion-fab class="scan">
+      <ion-fab-button @mousedown="scan" @mouseout="stopScan" >
+        <ion-icon :icon="scanOutline" />
+      </ion-fab-button>
+    </ion-fab>
+  </ion-content>
 </template>
 
 <script>
+import emitter from "@/event-bus"
 import { StreamBarcodeReader } from "vue-barcode-reader";
 import { IonButton, IonButtons, IonIcon, IonToolbar, modalController } from '@ionic/vue';
```

Comment:
Please import all the components used on the page, here `ion-fab`, `ion-content`, and `ion-fab-button` are missing.

Reply:
Done.